library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt

```diff
@@ -0,0 +1,17 @@
+This pull request is generated with proto changes between googleapis commit a17d4caf184b050d50cacf2b0d579ce72c31ce74 (exclusive) and 1a45bf7393b52407188c82e63101db7dc9c72026 (inclusive).
+Qualified commits are:
+[googleapis/googleapis@7a9a855](https://github.com/googleapis/googleapis/commit/7a9a855287b5042410c93e5a510f40efd4ce6cb1)
+[googleapis/googleapis@c7fd8bd](https://github.com/googleapis/googleapis/commit/c7fd8bd652ac690ca84f485014f70b52eef7cb9e)
+BEGIN_NESTED_COMMIT
+feat: [document-ai] expose model_type in v1 processor, so that user can see the model_type after get or list processor version
```

Comment:
Can we add a test case for a commit that contains multiple conventional commit messages? Like this [one](https://github.com/googleapis/googleapis/commit/aa16fdad909bc33e2d4ff04cfde56a46d0e52b13)? I think the current logic that adds library name may not take this scenario into consideration.

Reply:
Done.